### ISSUE TITLE
Character problem

### ISSUE THREAD

**Issue by Syntheq** [ikkentim/SampSharp#213]
When I type turkish character on chat, it disappears and when I type only one turkish character on chat, server does crash.

I am using
`codepage cp1254`

Typing "Kırılgan" and player text like that,
`Krlgan`

Typing 'ş' and server crash like that,
```
[02/08/2017 21:34:11] Exception thrownOnPlayerText:
System.IndexOutOfRangeException: Index was outside the bounds of the array.
  at ***.***.Player.OnText (SampSharp.GameMode.Events.TextEventArgs e) [0x000aa] in <d554937b11e147e29edfe4ae54a7701c>:0 
  at SampSharp.GameMode.Controllers.BasePlayerController+<>c.<RegisterEvents>b__0_5 (System.Object sender, SampSharp.GameMode.Events.TextEventArgs args) [0x0000a] in <da4ae371ce1a4d5da9a337d653eaf03e>:0 
  at SampSharp.GameMode.BaseMode.OnPlayerText (SampSharp.GameMode.World.BasePlayer player, SampSharp.GameMode.Events.TextEventArgs e) [0x0000a] in <da4ae371ce1a4d5da9a337d653eaf03e>:0 
  at SampSharp.GameMode.BaseMode.OnPlayerText (System.Int32 playerid, System.String text) [0x0000e] in <da4ae371ce1a4d5da9a337d653eaf03e>:0 
```

**Comment by ikkentim** [comment]
Show your code in Player.OnText please, since that is where it crashes

**Comment by Syntheq** [comment]
I found the problem. When I type "şşşşşş" on chat from game, e.Text is null.
Why turkish characters doesn't support on chat?
Dialogs and clientmessages with turkish characters works perfectly but not work on chat.

From Game,
https://prnt.sc/g3o2ag

From Console
`
[join] rootx has joined the server (0:127.0.0.1)
rootx => test (4)
rootx => krlgan (6)
rootx =>  (0)
[part] rootx has left the server (0:1)
`

Codes
`        public override void OnText(TextEventArgs e)
        {
            Console.WriteLine($"{Name} => {e.Text} ({e.Text.Count()})");
            base.OnText(e);
            return;
}
`

**Comment by ikkentim** [comment]
I'll  try fix this for the next version of sampsharp

**Comment by ikkentim** [comment]
This should be fixed in the newly released 0.8.0-alpha1 version. Please note this is a prerelease that should not yet be used on a production/public server.

**Comment by ikkentim** [comment]
Also closing this as a duplicate of #190
